mmv1/products/bigquerydatatransfer/Config.yaml

```diff
@@ -172,6 +181,18 @@ properties:
       reingests data for [today-10, today-1], rather than ingesting data for
       just [today-1]. Only valid if the data source supports the feature.
       Set the value to 0 to use the default value.
+  - !ruby/object:Api::Type::NestedObject
+    name: 'encryptionConfiguration'
+    description: |
```

Comment:
To be future-proof, can we make this description more generic and not specific about encryption key?

Reply:
updated description to match that found in API Reference.

Reply:
According to https://cloud.google.com/bigquery/docs/reference/datatransfer/rest/v1/projects.locations.transferConfigs#encryptionconfiguration - "The name of the KMS key used for encrypting BigQuery data." is the API description of `kmsKeyName`. Description of `EncryptionConfiguration` is "Represents the encryption configuration for a transfer."

Reply:
apologies seems like i got the descriptions mixed up, should be correct now.